ResumptionScenarios.md

```diff
@@ -0,0 +1,108 @@
+
+```
+Client                                                                                           | Server 
+1                                                                                                |
+2                                                                                                |
+3  -> SETUP(ResumeFlag(1))                                                                       |
+4                                                                                                | -> SETUP
+5  -> REQUEST_STREAM  (StreamID(1)) // RPosition(0) Position(1) IPosition(0)                     |
+6                                                                                                | -> REQUEST_STREAM  (Resume(1) StreamID(1)) // RPosition(0) Position(0) IPosition(1)
+7                                                                                                | <- PAYLOAD         (Resume(1) StreamID(1)) // RPosition(0) Position(1) IPosition(1)
+8  -> REQUEST_CHANNEL (StreamID(3)) // RPosition(0) Position(1) IPosition(0)                     |
+9                                                                                                | <- PAYLOAD         (Resume(1) StreamID(1)) // RPosition(0) Position(2) IPosition(1)
+10                                                                                               | -> REQUEST_CHANNEL (Resume(0) StreamID(3)) // RPosition(0) Position(0) IPosition(1)
+11                                                                                               | <- PAYLOAD         (Resume(1) StreamID(3)) // RPosition(0) Position(1) IPosition(1)
+12                                                                                               | <- REQUEST(N)      (Resume(0) StreamID(3)) // RPosition(0) Position(2) IPosition(1)
+13 <- PAYLOAD         (StreamID(1)) // RPosition(0) Position(1) IPosition(1)                     |
+14 -> KEEPALIVE       (NoE(1), { StreamID(1) : 1})                                               |
+15                                                                                               | -> KEEPALIVE       (NoE(1), { StreamID(1) : 1 })
+16                                                                                               | <- KEEPALIVE       (NoE(2), { StreamID(1) : 1, StreamID(3) : 1 })
+17 <- KEEPALIVE       (NoE(2), { StreamID(1) : 1, StreamID(3) : 1 })                             | 
+18 <- PAYLOAD         (Resume(1) StreamID(3)) // RPosition(1) Position(1) IPosition(1)           |  
+19 <- REQUEST(N)      (Resume(0) StreamID(3)) // RPosition(1) Position(1) IPosition(2)           |  
+20 -> REQUEST_STREAM  (StreamID(5)) // RPosition(0) Position(1) IPosition(0)                     |
+21~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~DISCON|
```

Comment:
This could be spelled out fully on both sides I think, it's clear that is it is the same event. Or perhaps also use directional arrows at the beginning of the line (like for payloads).